trytond/modules/__init__.py

```diff
@@ -368,9 +368,30 @@ def migrate_modules(cursor):
 
         def rename(cursor, table_name, old_name, new_name, var_name):
             table = Table(table_name)
-            cursor.execute(*table.update([getattr(table, var_name)],
+            fields = None
+            # If the view already exists in destination module
+            if table_name == 'ir_model_data':
+                fields = ['fs_id', 'model']
+            if table_name == 'ir_ui_view':
+                fields = ['model', 'name']
+            if fields:
+                query = ('DELETE from %(table)s where '
+                    '(%(fields)s) in ('
+                        'SELECT %(fields)s FROM %(table)s WHERE '
```

Comment:
python-sql

Reply:
j'ai essayé et pas réussi à faire un select in avec un ensemble de valeur.